src/test/regress/expected/replicate_reference_tables_to_coordinator.out

```diff
@@ -342,6 +370,25 @@ $Q$);
  f
 (1 row)
 
+TRUNCATE local_table;
+TRUNCATE numbers;
+NOTICE:  executing the command locally: TRUNCATE TABLE replicate_ref_to_coordinator.numbers_xxxxx CASCADE
+BEGIN;
+INSERT INTO local_table VALUES (1), (2), (3), (4);
+INSERT INTO numbers VALUES (1), (2), (3), (4);
+NOTICE:  executing the command locally: INSERT INTO replicate_ref_to_coordinator.numbers_8000001 AS citus_table_alias (a) VALUES (1), (2), (3), (4)
+ALTER TABLE numbers ADD COLUMN d int;
+NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (8000001, 'replicate_ref_to_coordinator', 'ALTER TABLE numbers ADD COLUMN d int;')
+SELECT * FROM local_table JOIN numbers USING(a);
```

Comment:
needs an ORDER BY

Reply:
I guess we should replace random function calls with something determistic in this test file as we can actually perform them now.

Reply:
I replaced them with a custom function my_random that returns a deterministic value.